docs/_rtd/windows_setup.md

```diff
@@ -0,0 +1,50 @@
+## Windows configuration
+
+Our repo works runs best on Linux-based systems (which includes Mac OS). If you need to run on Windows, you'll have some extra considerations to take into account. Note that this documentation applies for Windows 10 and Windows 11 machines.
+
+### WSL 2
+
+In order to download Docker, you need to install WSL 2 if you don't already have it. **We find it best to download and configure WSL 2 before anything else**. We recommend using the Command Prompt to run these commands (and later ones). 
```

Comment:
I would add a hyperlink to a wikipedia page or something of what the command prompt is